doc/source/quickstart_mxnet.rst

```diff
@@ -0,0 +1,279 @@
+        return train_data, val_data
+
+Define the training and loss with MXNet. The training of the dataset is done by looping over the dataset, measure the corresponding loss and optimize it. 
+
+.. code-block:: python
+
+    def train(net, train_data, epoch):
+        trainer = gluon.Trainer(net.collect_params(), "sgd", {"learning_rate": 0.03})
+        metric = mx.metric.Accuracy()
+        softmax_cross_entropy_loss = gluon.loss.SoftmaxCrossEntropyLoss()
+        for i in range(epoch):
+            train_data.reset()
+            for batch in train_data:
+                data = gluon.utils.split_and_load(
+                    batch.data[0], ctx_list=DEVICE, batch_axis=0
+                )
+                label = gluon.utils.split_and_load(
+                    batch.label[0], ctx_list=DEVICE, batch_axis=0
+                )
+                outputs = []
+                with ag.record():
+                    for x, y in zip(data, label):
+                        z = net(x)
+                        loss = softmax_cross_entropy_loss(z, y)
+                        loss.backward()
+                        outputs.append(z)
+                metric.update(label, outputs)
+                trainer.step(batch.data[0].shape[0])
+            name, acc = metric.get()
+            metric.reset()
+            print("training acc at epoch %d: %s=%f" % (i, name, acc))
+
+
+Define then the validation of the  machine learning network. We loop over the test set and measure the loss and accuracy of the test set. 
+
+.. code-block:: python
+
+    def test(net, val_data):
+        metric = mx.metric.Accuracy()
+        loss_metric = mx.metric.Loss()
+        loss = 0.0
+        val_data.reset()
+        for batch in val_data:
+            data = gluon.utils.split_and_load(batch.data[0], ctx_list=DEVICE, batch_axis=0)
+            label = gluon.utils.split_and_load(
+                batch.label[0], ctx_list=DEVICE, batch_axis=0
+            )
+            outputs = []
+            for x in data:
+                outputs.append(net(x))
+                loss_metric.update(label, outputs)
+                loss += loss_metric.get()[1]
+            metric.update(label, outputs)
+        print("validation acc: %s=%f" % metric.get())
+        print("validation loss:", loss)
+        accuracy = metric.get()[1]
+        return loss, accuracy
+
+After defining the training and testing of a MXNet machine learning model, we use the functions for the Flower clients.
+
+The Flower clients will use a simple Sequential model:
+
+.. code-block:: python
+
+    def main():
+        def model():
+            net = nn.Sequential()
+            net.add(nn.Dense(256, activation="relu"))
+            net.add(nn.Dense(10))
+            net.collect_params().initialize()
+            return net
+
+        train_data, val_data = load_data()
+
+        model = model()
+        init = nd.random.uniform(shape=(2, 784))
+        model(init)
+
```

Comment:
Maybe add a comment about the model initialization here?

Reply:
done